tests/core/chain-object/test_gas_estimation.py

```diff
@@ -14,49 +24,194 @@
+        # sha3 precompile
+        (b'\xff' * 32, None, ADDRESS_2, True, ByzantiumVM, 35333),
+        (b'\xff' * 32, None, ADDRESS_2, False, ByzantiumVM, 35345),
+        (b'\xff' * 320, None, ADDRESS_2, True, ByzantiumVM, 54840),
+        # 1000_tolerance binary search
+        (b'\xff' * 32, binary_gas_search_1000_tolerance, ADDRESS_2, True, ByzantiumVM, 23935),
+
+        (b'', None, ADDR_1010, True, ConstantinopleVM, 21000),
+        (b'', None, ADDR_1010, False, ConstantinopleVM, 21000),
+        (b'\xff' * 10, None, ADDR_1010, True, ConstantinopleVM, 21680),
+        (b'\xff' * 10, None, ADDR_1010, False, ConstantinopleVM, 21680),
+        # sha3 precompile
+        (b'\xff' * 32, None, ADDRESS_2, True, ConstantinopleVM, 35333),
+        (b'\xff' * 32, None, ADDRESS_2, False, ConstantinopleVM, 35345),
+        (b'\xff' * 320, None, ADDRESS_2, True, ConstantinopleVM, 54840),
+        # 1000_tolerance binary search
+        (b'\xff' * 32, binary_gas_search_1000_tolerance, ADDRESS_2, True, ConstantinopleVM, 23935),
+
+        (b'', None, ADDR_1010, True, PetersburgVM, 21000),
+        (b'', None, ADDR_1010, False, PetersburgVM, 21000),
+        (b'\xff' * 10, None, ADDR_1010, True, PetersburgVM, 21680),
+        (b'\xff' * 10, None, ADDR_1010, False, PetersburgVM, 21680),
+        # sha3 precompile
+        (b'\xff' * 32, None, ADDRESS_2, True, PetersburgVM, 35333),
+        (b'\xff' * 32, None, ADDRESS_2, False, PetersburgVM, 35345),
+        (b'\xff' * 320, None, ADDRESS_2, True, PetersburgVM, 54840),
+        # 1000_tolerance binary search
+        (b'\xff' * 32, binary_gas_search_1000_tolerance, ADDRESS_2, True, PetersburgVM, 23935),
+
+        (b'', None, ADDR_1010, True, IstanbulVM, 21000),
+        (b'', None, ADDR_1010, False, IstanbulVM, 21000),
+        (b'\xff' * 10, None, ADDR_1010, True, IstanbulVM, 21160),
+        (b'\xff' * 10, None, ADDR_1010, False, IstanbulVM, 21160),
         # sha3 precompile
-        (b'\xff' * 32, None, ADDRESS_2, True, 35333),
-        (b'\xff' * 32, None, ADDRESS_2, False, 35345),
-        (b'\xff' * 320, None, ADDRESS_2, True, 54840),
+        (b'\xff' * 32, None, ADDRESS_2, True, IstanbulVM, 33675),
+        (b'\xff' * 32, None, ADDRESS_2, False, IstanbulVM, 33687),
+        (b'\xff' * 320, None, ADDRESS_2, True, IstanbulVM, 38265),
         # 1000_tolerance binary search
-        (b'\xff' * 32, binary_gas_search_1000_tolerance, ADDRESS_2, True, 23935),
+        (b'\xff' * 32, binary_gas_search_1000_tolerance, ADDRESS_2, True, IstanbulVM, 22272),
     ),
     ids=[
-        'simple default pending',
-        'simple default',
-        '10 bytes default pending',
-        '10 bytes default',
-        'sha3 precompile 32 bytes default pending',
-        'sha3 precompile 32 bytes default',
-        'sha3 precompile 320 bytes default pending',
-        'sha3 precompile 32 bytes 1000_tolerance binary pending',
+        'simple default pending for FrontierVM',
```

Comment:
I think that with such a large list, we should inline the ids using this format:

```python
pytest.param(
            b'', None, ADDR_1010, True, FrontierVM, 21000, id='simple default pending for FrontierVM',
        ),
```